src/olympia/versions/models.py

```diff
@@ -166,7 +166,7 @@ class Version(OnChangeMixin, ModelBase):
     addon = models.ForeignKey(
         'addons.Addon', related_name='versions', on_delete=models.CASCADE)
     license = models.ForeignKey(
-        'License', null=True, on_delete=models.CASCADE)
+        'License', null=True, blank=True, on_delete=models.CASCADE)
```

Comment:
I am surprised that this change does not require a migration.

Reply:
Does https://docs.djangoproject.com/en/3.1/ref/models/fields/#django.db.models.Field.blank help?

Reply:
No. Please run `./manage.py makemigrations versions` and edit the generated migration to only add the `AlterField` for the `license` field.

also cc @eviljeff, there are other operations (one of them being to use a `VersionStringField`

Reply:
🤷‍♂️ the VersionStringField changes are no-ops at a db level

Reply:
@willdurand can you clarify what the expected migration should be? I ran `makemigrations` and the generated migrations file does not contain any changes related to patch:

```
# Generated by Django 2.2.16 on 2020-10-28 14:17

from django.db import migrations
import olympia.versions.fields


class Migration(migrations.Migration):

    dependencies = [
        ('versions', '0012_auto_20201020_1147'),
    ]

    operations = [
        migrations.RemoveField(
            model_name='version',
            name='recommendation_approved',
        ),
        migrations.AlterField(
            model_name='version',
            name='version',
            field=olympia.versions.fields.VersionStringField(default='0.1', max_length=255),
        ),
    ]
```

Reply:
Interesting. Here is the one I got:

```python
  # Generated by Django 2.2.16 on 2020-10-28 13:55

  from django.db import migrations, models
  import django.db.models.deletion
  import olympia.versions.fields


  class Migration(migrations.Migration):

      dependencies = [
          ('versions', '0012_auto_20201020_1147'),
      ]

      operations = [
          migrations.RemoveField(
              model_name='version',
              name='recommendation_approved',
          ),
          migrations.AlterField(
              model_name='version',
              name='license',
              field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, to='versions.License'),
          ),
          migrations.AlterField(
              model_name='version',
              name='version',
              field=olympia.versions.fields.VersionStringField(default='0.1', max_length=255),
          ),
      ]
```

Reply:
@eviljeff do you know when `recommendation_approved` was removed btw?

Reply:
https://github.com/mozilla/addons-server/commit/ca34f9c04e38dbf8f25e63e7ae074136bc4bf37e#diff-fdb00a545bc5f3b6d17bc710cde29232c9faee776dacf41c2448090cb6917b69